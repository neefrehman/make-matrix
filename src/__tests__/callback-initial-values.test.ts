import makeMatrix from "../index";

// Test arrays of specific dimensions, with callbacks returning initialised values
test("length: 1 test - With string callback initialValues", () => {
    expect(makeMatrix([1], () => "string")).toMatchSnapshot();
});

test("length: 2 test - With math callback initialValues", () => {
    expect(makeMatrix([2], () => 10 - 2)).toMatchSnapshot();
});
