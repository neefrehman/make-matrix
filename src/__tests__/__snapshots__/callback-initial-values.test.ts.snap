// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`length: 1 test - With string callback initialValues 1`] = `
Array [
  "string",
]
`;

exports[`length: 2 test - With math callback initialValues 1`] = `
Array [
  8,
  8,
]
`;
